# This file was autogenerated using flake8.
pythinux.py:1:1: E265 block comment should start with '# '
pythinux.py:6:1: F401 'traceback' imported but unused
pythinux.py:12:1: F401 'threading' imported but unused
pythinux.py:15:1: F401 'asyncio' imported but unused
pythinux.py:25:1: E722 do not use bare 'except'
pythinux.py:36:33: E261 at least two spaces before inline comment
pythinux.py:37:36: E261 at least two spaces before inline comment
pythinux.py:38:36: E261 at least two spaces before inline comment
pythinux.py:39:45: E261 at least two spaces before inline comment
pythinux.py:39:80: E501 line too long (104 > 79 characters)
pythinux.py:40:31: E261 at least two spaces before inline comment
pythinux.py:41:30: E261 at least two spaces before inline comment
pythinux.py:50:80: E501 line too long (86 > 79 characters)
pythinux.py:151:80: E501 line too long (85 > 79 characters)
pythinux.py:168:80: E501 line too long (82 > 79 characters)
pythinux.py:202:80: E501 line too long (84 > 79 characters)
pythinux.py:255:80: E501 line too long (85 > 79 characters)
pythinux.py:256:80: E501 line too long (82 > 79 characters)
pythinux.py:271:80: E501 line too long (103 > 79 characters)
pythinux.py:357:80: E501 line too long (109 > 79 characters)
pythinux.py:365:80: E501 line too long (109 > 79 characters)
pythinux.py:413:80: E501 line too long (82 > 79 characters)
pythinux.py:514:80: E501 line too long (83 > 79 characters)
pythinux.py:547:80: E501 line too long (106 > 79 characters)
pythinux.py:633:80: E501 line too long (86 > 79 characters)
pythinux.py:634:80: E501 line too long (84 > 79 characters)
pythinux.py:640:80: E501 line too long (82 > 79 characters)
pythinux.py:642:80: E501 line too long (124 > 79 characters)
pythinux.py:654:80: E501 line too long (86 > 79 characters)
pythinux.py:734:80: E501 line too long (88 > 79 characters)
pythinux.py:891:80: E501 line too long (85 > 79 characters)
pythinux.py:897:5: E266 too many leading '#' for block comment
pythinux.py:901:5: E266 too many leading '#' for block comment
pythinux.py:908:80: E501 line too long (93 > 79 characters)
pythinux.py:912:5: E266 too many leading '#' for block comment
pythinux.py:943:80: E501 line too long (99 > 79 characters)
pythinux.py:944:80: E501 line too long (102 > 79 characters)
pythinux.py:954:5: E301 expected 1 blank line, found 0
pythinux.py:957:1: E302 expected 2 blank lines, found 1
pythinux.py:958:80: E501 line too long (86 > 79 characters)
pythinux.py:961:1: E302 expected 2 blank lines, found 1
pythinux.py:974:80: E501 line too long (90 > 79 characters)
pythinux.py:976:5: F841 local variable 'current_directory' is assigned to but never used
pythinux.py:1062:1: W293 blank line contains whitespace
pythinux.py:1078:21: F601 dictionary key 'saveUserList' repeated with different values
pythinux.py:1085:21: F601 dictionary key 'saveUserList' repeated with different values
pythinux.py:1128:5: F841 local variable 'CURRDIR' is assigned to but never used
pythinux.py:1134:80: E501 line too long (80 > 79 characters)
pythinux.py:1282:5: F841 local variable 'current_directory' is assigned to but never used
pythinux.py:1307:80: E501 line too long (83 > 79 characters)
pythinux.py:1312:80: E501 line too long (80 > 79 characters)
pythinux.py:1337:80: E501 line too long (87 > 79 characters)
pythinux.py:1352:80: E501 line too long (87 > 79 characters)
pythinux.py:1376:9: F841 local variable 'x' is assigned to but never used
pythinux.py:1566:5: F841 local variable 'g' is assigned to but never used
